bin/not-in-stable.go

```diff
@@ -5,12 +5,12 @@ import (
 	"bytes"
 	"flag"
 	"fmt"
-	"log"
 	"os"
 	"os/exec"
 	"regexp"
 
 	"github.com/coreos/go-semver/semver"
+	"github.com/rclone/rclone/fs"
```

Comment:
@ncw Maybe it will not be correct to use fs. in this file, so it needs to be excluded, or?

Reply:
Yes probably exclude!